app/views/api/drops/index.json.jbuilder

```diff
@@ -0,0 +1,13 @@
+json.array! @drops do |drop|
+  
+  json.id drop.id
+  json.title drop.title
+  json.sc_track drop.sc_track
+  json.place do
+    json.id drop.place.id
+    json.name drop.place.name
+    json.latitude drop.place.latitude
+    json.longitude drop.place.longitude
+  end
+end
+
```

Comment:
Same story here: As long as whitespace has the function of improving readability I would keep them, but I'd suggest dropping the whitespace line at the end here.